src/locale/lang/id.js

```diff
@@ -0,0 +1,86 @@
+export default {
+  el: {
+    datepicker: {
+      now: 'Sekarang',
+      today: 'Hari ini',
```

Comment:
I'm afraid there's no equal word other than `hari ini`. But may be we can use shorted word of `sekarang` which is `Skrng`. It's not usual but I guess people will got it.